src/Symfony/Component/VarDumper/VarDumper.php

```diff
@@ -115,4 +116,18 @@ private static function getDefaultContextProviders(): array
             'source' => new SourceContextProvider(null, null, $fileLinkFormatter),
         ];
     }
+
+    private static function guessMostSuitableDumper(): DataDumperInterface
+    {
+        if (\in_array(\PHP_SAPI, ['cli', 'phpdbg', 'embed'], true)) {
+            return new CliDumper();
+        }
+
+        $acceptHeader = $_SERVER['HTTP_ACCEPT'] ?? null;
+        if (null === $acceptHeader || str_contains($acceptHeader, 'text/html') || str_contains($acceptHeader, '*/*')) {
```

Comment:
not sure why we should use html when `*/*` is found but not `html`
```suggestion
        if (str_contains($_SERVER['HTTP_ACCEPT'] ?? 'html', 'html')) {
```

Reply:
If the header is present and we reach this condition, we are necessarily in a SAPI context other than CLI. So I'm thinking that it's still worth rendering the HTML if the client sends "accept all". It seems more interesting to me than using CliDumper with the various escape characters that won't really work outside of a terminal. What do you think?

Reply:
This is used by Chrome dev tools

Reply:
Alright, updated :+1: